bin/citgm.js

```diff
@@ -70,40 +75,42 @@ if (!citgm.windows) {
 
 app.parse(process.argv);
 
+var config = app.config ? util._extend(require(resolve(process.cwd(), app.config)), app) : app;
+
 var log = logger({
-  level:app.verbose,
-  nocolor: !app.color
+  level:config.verbose,
+  nocolor: !config.color
 });
 
 update(log);
 
-if (!app.su) {
+if (!config.su) {
   require('root-check')(); // silently downgrade if running as root...
                            // unless --su is passed
 } else {
   log.warn('root', 'Running as root! Use caution!');
 }
 
 if (!mod) {
-  app.outputHelp();
+  config.outputHelp();
```

Comment:
I wanted to ensure CLI params took precedence over the config file. I suppose that can be done more cleanly, keeping the `app` object. Stay tuned for a second, cleaner, pass at this.